src/utils/GitHub.test.js

```diff
@@ -1,6 +1,11 @@
-import {getDownloadURL, parseGitHubRepositoryURL} from './GitHub'
+import {
+  getDownloadURL,
+  parseGitHubRepositoryURL,
+  postIssue,
+} from './GitHub'
 
 
+const created = 201
```

Comment:
Please rename this variable to be consistent with the other HTTP status code variables, e.g. httpCreated.